monitoring/uss_qualifier/scd/simulator/utils.py

```diff
@@ -0,0 +1,104 @@
+from pathlib import Path
+from typing import List, Union, Literal
+from geojson import Feature
+import enum
+from shapely.geometry import LineString, Polygon
+from monitoring.monitorlib.typing import ImplicitDict, StringBasedDateTime
+from monitoring.monitorlib.scd import Altitude, Volume4D,  OperationalIntentState
+from pathlib import Path
+
+class OutputSubDirectories(ImplicitDict):
+    ''' A class to hold information about output directories that will be generated when writing the files to disk. '''
+    injection_payloads: Path 
+    scd_rules: Path
+
+class InjectionStatus(str, enum.Enum):
```

Comment:
This seems to be mirroring [the SCD automated testing API](https://github.com/interuss/dss/blob/d90153e203dd917fb57a5ab269146095567a64ab/interfaces/automated-testing/scd/scd.yaml#L441) -- let's follow [the RID style](https://github.com/interuss/dss/tree/master/monitoring/monitorlib/rid_automated_testing) and put all the data types mirrored from the automated testing API in monitorlib/scd_automated_testing

Reply:
OK thanks, I have now created the file and moved this to that file.